schemas/dswx_hls.yaml

```diff
@@ -0,0 +1,35 @@
+runconfig:
+    name: str()
+
+    groups:
+        pge_name_group:
+            pge_name: enum('DSWX_HLS_PGE')
+
+        input_file_group:
+            # REQUIRED - Single HLS dataset (directory or files)
+            input_file_path: list(str(), min=1)
+       
+        dynamic_ancillary_file_group:
+            # Digital elevation model
+            dem_file: include(str(), required=False)
+
+            # Land cover map
+            landcover_file: include(str(), required=False)
+
+            # Built up land map
+            built_up_cover_fraction_file: include(str(), required=False)
+       
+        product_path_group:
+            # Directory where PGE will place results
+            product_path: str()
+       
+            # Directory where SAS can write temporary data
+            scratch_path: str()
+       
+            # Intermediate file name.  SAS writes the output product to the following file.
+            # After the SAS completes, the PGE wrapper renames the product file
+            # according to proper file naming conventions.
+            sas_output_file: str()
+       
+        primary_executable:
+            product_type: enum('DSWX_HLS')
```

Comment:
In my opinion, the schema should have a processing section hosting all the parameters that the user would like to tune.